### DISCUSSION TITLE
Disable physical switch

### DISCUSSION THREAD

**Discussion by jokerigno** [arendst/Tasmota#21323] [Q&A]
Hi , I have several Shelly 1 flashed with tasmota. I have also 3 daughters and the youngest has started joking with switches. I was wondering if there’s a way to exclude the physical switch from console . So only voice could be use to switch lights. Any suggestions?

**Comment by sfromis** [comment]
As I told you on Discord, you can unconfigure the input, which is inside Tasmota, not physically.

**Reply by jokerigno** [reply]
What is the command in console to disable input? Maybe Poweronstate 4?

**Reply by sfromis** [reply]
The `Template` command allows you to replace an existing template config.

Assuming standard Tasmota config for the device, you can also decouple Switch input from Relay actions using the command `SetOption114`. Then you still get MQTT messages.
